### PR TITLE
Theme Rebrand

### PR DESCRIPTION
**Pull request by giovannism20**
This PR includes a lot modifications such as:
- Add the structure to have blog
- Removing unnecessary comments
- Update on composer.json and README.md
- Split some "components"
- Remove unused files

### CODE REVIEW

File: source/_layouts/main.blade.php [hunk 1]
```diff
@@ -1,15 +1,42 @@
...
+      href="assets/images/favicon.svg"
+      type="image/svg"
+    />
+
+    <!-- ===== All CSS files ===== -->
+    <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="assets/css/animate.css" />
+    <link rel="stylesheet" href="assets/css/lineicons.css" />
+    <link rel="stylesheet" href="assets/css/ud-styles.css" />
+  </head>
```
Comment (vitormattos) [inline source/_layouts/main.blade.php:45]:
```suggestion
  </head>
  <body>
      @yield('body')
  </body>
```

File: source/_layouts/main.blade.php [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
-    </head>
-    <body class="text-gray-900 font-sans antialiased">
-        @yield('body')
-    </body>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LibreSign - Electronic signature of digital documents</title>
```
Comment (vitormattos) [inline source/_layouts/main.blade.php]:
```suggestion
    <title>{{ $page->title }}</title>
```

File: source/_layouts/main.blade.php [hunk 3]
```diff
@@ -1,15 +1,42 @@
...
-        <link rel="canonical" href="{{ $page->getUrl() }}">
-        <meta name="description" content="{{ $page->description }}">
-        <title>{{ $page->title }}</title>
-        <link rel="stylesheet" href="{{ mix('css/main.css', 'assets/build') }}">
-        <script defer src="{{ mix('js/main.js', 'assets/build') }}"></script>
-    </head>
-    <body class="text-gray-900 font-sans antialiased">
-        @yield('body')
-    </body>
+<html lang="en">
```
Comment (vitormattos) [inline source/_layouts/main.blade.php]:
```suggestion
<html lang="{{ $page->language ?? 'en' }}">
```

File: source/_layouts/main.blade.php [hunk 4]
```diff
@@ -1,15 +1,42 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LibreSign - Electronic signature of digital documents</title>
+
+    <!-- Primary Meta Tags -->
+    <meta name="title" content="LibreSign - Electronic signature of digital documents">
+    <meta name="description" content="LibreSign - Electronic signature of digital documents">
```
Comment (vitormattos) [inline source/_layouts/main.blade.php]:
```suggestion
    <link rel="canonical" href="{{ $page->getUrl() }}">
    <meta name="description" content="{{ $page->description }}">
```

File: source/_layouts/main.blade.php [hunk 5]
```diff
@@ -1,15 +1,42 @@
...
+      rel="shortcut icon"
+      href="assets/images/favicon.svg"
+      type="image/svg"
+    />
+
+    <!-- ===== All CSS files ===== -->
+    <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="assets/css/animate.css" />
+    <link rel="stylesheet" href="assets/css/lineicons.css" />
+    <link rel="stylesheet" href="assets/css/ud-styles.css" />
```
Comment (vitormattos) [inline source/_layouts/main.blade.php:41]:
```suggestion
    <link rel="stylesheet" href="assets/css/ud-styles.css" />
    <link rel="stylesheet" href="{{ mix('css/main.css', 'assets/build') }}">
    <!-- Primary Meta Tags -->
    <script defer src="{{ mix('js/main.js', 'assets/build') }}"></script>
```

**Review by vitormattos — CHANGES_REQUESTED** [review]
Small fixes and able to merge and follow making small PR's.

**Comment by vitormattos** [comment]
DCO disliked your commit.

File: docker-compose.yml [hunk 6]
```diff
@@ -1,7 +1,9 @@
 services:
 
   backend:
-    build: .docker
+    build:
+      context: docker
+      dockerfile: Dockerfile
```
Comment (vitormattos) [inline docker-compose.yml]: Only is necessary if the Dockerfile have a name different as `Dockerfile`
```suggestion
```

**Comment by giovannism20** [comment]
Hey @vitormattos since do you suggested for make little modifications from this pull request I'll not make more commits here ;)

**Review by vitormattos — APPROVED** [review]

**Pull request merged**